src/Microsoft.Owin.Security.Authorization/AuthorizationHelper.cs

```diff
@@ -32,53 +29,25 @@ public AuthorizationHelper(IOwinContextAccessor owinContextAccessor)
                 throw new ArgumentNullException(nameof(authorizeAttribute));
             }
 
-            AuthorizationOptions options;
+            AuthorizationDependencies dependencies;
             if (controller != null)
             {
-                options = controller.AuthorizationOptions;
+                var dependenciesProvider = AuthorizationDependenciesProvider.CreateDefault();
+                dependencies = dependenciesProvider.OnCreate?.Invoke(controller.AuthorizationOptions, null);
```

Comment:
as the project is called Owin-Authorization I was not sure if we should support this or not - probably necessary to extend authzcontroller interface then